Documentation/planning/multi-sdk-band-support.md

```diff
@@ -0,0 +1,258 @@
+4. The input artifacts for components **not** built in the the product source build. Examples include:
+   - Prebuilts
+   - In a subsetted VMR case, components built in *another* VMR build.
+
+***Note: When dealing with just a 1xx SDK, full build, with no prebuilts, #4 is an empty set, and #2 and #3 are the previously-source-built artifacts passed with --with-packages.***
+
+When building a VMR, a distro maintainer will combine the sets of artifacts from #2, #3 and #4 into a single directory and pass it with --with-artifacts. Below, distro maintainer workflow scenarios are presented. The following abbreviations are used:
+
+- PSB - Previously source-built (from a previous servicing/preview iteration, or from a bootstrap build)
+- CSB - Current source-built (just built, e.g. from a 1xx VMR build).
+
+### Previews and major release GA builds
+
+**No Change**
+
+A distro maintainer sees no change here. Until a few months **after** GA, only the 1xx band ships. The workflow remains the same as today. Clone the VMR branch/tag in question, build using previously source-built artifacts and SDKs.
+
+### A distro maintainer wishing to only support the 1xx SDK
+
+**No change**
+
+### A distro maintainer wishing to build the initial release (n00) of a 2xx+ band SDK
+
+The initial release of a 2xx+ band SDK should be straightforward. It is unlikely (though possible) that the initial 2xx band would depend on a 2xx SDK to build. If this is not the case, then a distro maintainer performs the following actions, and no bootstrap will be required:
+1. **Checkout** 1xx branch:
+2. **Build** using `./build.sh --with-sdk <PSB 1xx SDK> --with-artifacts <PSB 1xx artifacts>`.
+3. **Gather** 1xx build outputs from `artifacts/x64/Release` = `<CSB 1xx artifacts>`.
+4. **Prepare** 2xx inputs - `<PSB 1xx artifacts> + <CSB 1xx artifacts>`
+5. **Checkout** 2xx branch
+6. **Build** using `./build.sh --with-sdk <PSB 1xx SDK> --with-artifacts <2xx inputs>`
+
+If a 2xx SDK is required to build the initial 2xx SDK, then the following workflow is used:
+
+1. **Checkout** 1xx branch:
+2. **Build** using `./build.sh --with-sdk <PSB 1xx SDK> --with-artifacts <PSB 1xx artifacts>`.
+3. **Gather** 1xx build outputs from `artifacts/x64/Release` = `<CSB 1xx artifacts>`.
+4. **Prepare** 2xx bootstrap inputs - `<PSB 1xx artifacts> + <CSB 1xx artifacts>`
+5. **Checkout** 2xx branch
+6. **Build** using `./build.sh --with-artifacts <2xx bootstrap inputs>`
+7. **Prepare** 2xx inputs - `<PSB 1xx artifacts> + <CSB 1xx artifacts> + <CSB 2xx artifacts>`
+8. **Build** using `./build.sh --with-sdk <CSB 2xx SDK> --with-artifacts <2xx inputs>`
+
+### A distro maintainer wishing to support only a 2xx SDK (monthly workflow)
+
+A distro maintainer who only wants to support the 2xx SDK still needs to build the 1xx branch to obtain the runtime and an updated 1xx SDK for use the next month. In this case, we build the 2xx inputs with the 2xx SDK.
+
+1. **Checkout** 1xx branch:
```

Comment:
Not sure it's really needed but if we used a mermaid flowchart and for example made PSB round and CSB rectangles, and we coloured the bands different colours, maybe it would be super easy to read?

Reply:
I added one to an appendix